CMakeLists.txt

```diff
@@ -86,6 +86,15 @@ if(NOT MSVC)
     set_directory_properties(PROPERTIES INCLUDE_DIRECTORIES ${DL_INCLUDE_DIR})
     add_definitions(-DHAVE_DL=1)
   endif()
+else()
+  option(MSVC_USE_MP "Set to ON to build with the /MP option (Visual Studio 2005 and above)." ON)
```

Comment:
Do we really need a configurable option?
Why not just always add /MP for VS2005 or later?